libs/clients/hms-loans/src/lib/hmsLoans.provider.ts

```diff
@@ -33,6 +33,7 @@ export const HmsLoansApiProvider: Provider<LibraApi> = {
                 scope: [HmsScope.loans],
               }
             : undefined,
+          timeout: 60000,
```

Comment:
wow, why the extreme timeout?

Reply:
ugh, the service is very slow on dev/staging. They say it's much faster on prod.